site/static/js/plugins.js

```diff
@@ -43,6 +43,23 @@ function convertFromHugo(content) {
   return content.replace(hugoHideDirectives, "").replace(hugoDirectiveTags, "");
 }
 
+async function addVersionDropdown() {
+  const sidebar = document.getElementsByClassName("sidebar").item(0);
+  const versionDropdown = `
+  <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Version: 1.0.0-alpha.3
```

Comment:
There needs to be a fast follow to properly version the docs in the CD pipeline. There are not multiple pages that need to be manually updated whenever there's a release.